cmake/common/CompileSettingsDebug.cmake

```diff
@@ -30,16 +30,16 @@ include(CMakePackageConfigHelpers)
 #
 if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
     if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR ${CMAKE_COMPILER_IS_GNUCXX} )
-        target_compile_options(CompileSettingsDebug INTERFACE -O0 -g -Wextra)
+        target_compile_options(CompileSettingsDebug INTERFACE -O0 -g)
```

Comment:
Perhaps I'm missing something but should these not be in line with the options in release?

Reply:
This config file is specific to Debug mode